### ISSUE TITLE
18 Conduct thorough test of message, contacts and feedback systems including the ability to reply

### ISSUE THREAD

**Issue by Captainkirkdawson** [FreeUKGen/MyopicVicar#1627]
There is a need to ensure that all breadcrumbs, actions and reports are correct in light of the addition of the reply to system

Use priority of 1380

depends on #1380

**Comment by edickens** [comment]
I have just done a reply to the message "Feedback" system.  It worked, but I don't know who has looked at it.  There is just my email address in the header.  From me to me.  And the subject line is not in the reply.

![capture](https://user-images.githubusercontent.com/8876165/47024824-5f3e9980-d15a-11e8-8b11-4837dabefe70.JPG)

**Comment by Captainkirkdawson** [comment]
I would prefer to see this story focus on conducting systematic tests of all aspects of the systems to make sure that all components works together as expected. Any issues uncovered are to be written as separate stories.

**Comment by Captainkirkdawson** [comment]
@edickens It would appear that your test worked as expected and sesigned.

**Comment by SteveBiggs** [comment]
I received a copy of your reply @edickens

**Comment by Sherlock21** [comment]
i only received a copy of Post 2 above, and that came by ordinary email form eric D.
Nothing via FR messages on my FR home page.

**Comment by SteveBiggs** [comment]
I think all the issues under #1380 need to be addressed before we spend further time testing the message system.

**Comment by Captainkirkdawson** [comment]
Agreed; am currently working on those associated with #1380. Many are in the review status

**Comment by stsccfr** [comment]
Closing this as being too broad.
